### PR TITLE
Update show credentials documentation

### PR DESCRIPTION
**Pull request by loqo71la**

### CODE REVIEW

**Comment by loqo71la** [comment]
@JimmyRojas @marcocdlv please review my changes

File: _docs/credentials.md [hunk 1]
```diff
@@ -123,6 +125,20 @@ or
 
 	$ gradle showCredentials -Plocation=home
 
+#### ***Status***
+
+It is also possible to display the status of all credentials(*valid/invalid*) by the parameter ***status***
```
Comment (JimmyRojas) [inline _docs/credentials.md]: remove this

(_valid/invalid_)

File: _docs/credentials.md [hunk 2]
```diff
@@ -123,6 +125,20 @@ or
...
+
+	$ gradle showCredentials -Pstatus
+
+This parameter can filter credentials with two values: 'isValid' and 'isInvalid'.
```
Comment (JimmyRojas) [inline _docs/credentials.md]: use simple words..

This parameter can filter credentials with two values: 'isValid' and 'isInvalid'.

This parameter can filter credentials with two values: 'valid' and 'invalid'.

File: _docs/credentials.md [hunk 3]
```diff
@@ -244,4 +260,72 @@ Those credentials are located at /home/john/credentials.dat
...
+```bash
+:showCredentials
+*********************************************
+                Credentials
+*********************************************
+
+Id : john.enforce
+User name : john@enforce.com
+Type : Production/Developer
+Status : is valid
```
Comment (JimmyRojas) [inline _docs/credentials.md]: Status : Ok

File: _docs/credentials.md [hunk 4]
```diff
@@ -244,4 +260,72 @@ Those credentials are located at /home/john/credentials.dat
...
+
+Id : mine
+User name : john@hotmail.com
+Type : Production/Developer
+Status : is valid
+
+Id : my
+User name : john.cdlv@gmail.com
+Type : Production/Developer
+Status : is invalid - Invalid username, password, security token; or user locked out.
```
Comment (JimmyRojas) [inline _docs/credentials.md]: Status : Outdated - Invalid username, password, security token; or user locked out.

**Comment by JimmyRojas** [comment]
please see some comments..

**Comment by loqo71la** [comment]
@JimmyRojas I completed your observations, please review my changes

File: _docs/credentials.md [hunk 5]
```diff
@@ -222,21 +238,91 @@ Output:
                 Credentials
 *********************************************
 
-Id : john.enforce
+Id        : john.enforce
```
Comment (JimmyRojas) [inline _docs/credentials.md]: Is not possible use a tabulator??

\t

**Comment by JimmyRojas** [comment]
looks good to me.. :)

**Comment by naverus** [comment]
@JimmyRojas, @dan-canqui  great job guys

**Comment by naverus** [comment]
From my point of view, when the tool shows all credentials by status it would be better to have all credentials grouped by status and does not add the status by each credential, because sometimes you could have a bunch of valid/invalid credentials and it would be hard to review all of them(EG I had more than 32 accounts :p), so below you have an example.

---

```
             Credentials
```

---

*********\* Valid Credentials **************
 Id : john.enforce
 User name : john@enforce.com
 Type : Production/Developer

 Id : mine
 User name : john@hotmail.com
 Type : Production/Developer

*********\* Invalid Credentials *************\* 
 Id : my
 User name : john.cdlv@gmail.com
 Type : Production/Developer
 Caused By: Invalid username, password, security token; or user locked out.

 Id : recru
 User name : admin@enforce.com
 Type : dev (Specified)
 Caused By: Invalid username, password, security token; or user locked out. 

In this example you can see that the status is not duplicated and looks cleaner that the current one, please take in consideration this suggestion.

JIC if you want I could participate in the features definition, this is my skype account navor.nunez

**Comment by ziur** [comment]
@naverus, Glad to 'read' from you again. 
We also had the same idea but because of time constraints we decided to keep it as is right now, and later provide grouping functionality.

**Comment by naverus** [comment]
@ziur Cool...

**Pull request merged**